data/strings/strings.txt

```diff
@@ -16619,7 +16621,7 @@
 
   [big_font]
     tags = android,ios
-    en = Increase font size on the map
+    en = Increase map label size
```

Comment:
Agreed it is less clear.

`en = Increase font size for map labels`

Reply:
tweaked to 'Increase size for map labels', as font isn't really correct and text is also implied by 'labels'

Reply:
That's great and will be understood.